mlflow/tracing/clients/local.py

```diff
@@ -66,3 +66,27 @@ def get_trace(self, request_id: str) -> Optional[Trace]:
 
     def _flush(self):
         self.queue.clear()
+
+
+class InMemoryTraceClientWithTracking(InMemoryTraceClient):
```

Comment:
nit, not a blocker: Maybe we wanna have this in other file like `clients/databricks.py` as it's not longer local.

Reply:
hmm maybe tracking.py?

Reply:
or remote.py